### PR TITLE
docs(recipes): Calculating average and percentiles

### PR DESCRIPTION
**Pull request by igorlukanin**
Recreated PR.

### CODE REVIEW

File: docs/content/Examples-Tutorials-Recipes/Calculating-Average-and-Percentiles.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+subCategory: Data schema
+menuOrder: 1
+---
+
+## Use case
+
+We want to understand the distribution of values for a certain numeric property
+within a dataset. We're used to average values and intuitively understand how to
+calculate them. However, we also know that average values can be misleading for
+[skewed](https://en.wikipedia.org/wiki/Skewness) distributions which are frequent in
```
Comment (rpaik) [inline docs/content/Examples-Tutorials-Recipes/Calculating-Average-and-Percentiles.md:14]:
```suggestion
[skewed](https://en.wikipedia.org/wiki/Skewness) distributions which are common in
```

File: docs/content/Examples-Tutorials-Recipes/Calculating-Average-and-Percentiles.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+the real world: for example, 2.5 is the average value for both `(1, 2, 3, 4)` and
+`(0, 0, 0, 10)`.
+
+So, it's usually better to use [percentiles](https://en.wikipedia.org/wiki/Percentile).
+Parameterized by a fractional number `n = 0..1`, the n-th percentile equals to a value
```
Comment (rpaik) [inline docs/content/Examples-Tutorials-Recipes/Calculating-Average-and-Percentiles.md:19]:
```suggestion
Parameterized by a fractional number `n = 0..1`, where the n-th percentile is equals to a value
```

**Comment by rpaik** [comment]
Just a couple of minor suggestions 😄

**Comment by igorlukanin** [comment]
Thanks, @rpaik!

**Comment by rchkv** [comment]
LGTM! 👍

**Pull request merged**